govuk_publishing_components.gemspec

```diff
@@ -26,8 +26,8 @@ Gem::Specification.new do |s|
   s.add_dependency "sassc-rails", ">= 2.0.1"
   s.add_dependency "sprockets", "~> 3.7.2"
 
-  s.add_development_dependency "capybara", "~> 3.25"
-  s.add_development_dependency "foreman", "~> 0.85"
+  s.add_development_dependency "capybara", "= 3.25"
```

Comment:
Do we need to do this?

Reply:
Unfortunately, yes 😞otherwise CI will fail